src/components/IndexScreen.js

```diff
@@ -35,11 +31,11 @@ class IndexScreen extends PureComponent {
   render() {
     return (
       <View style={styles.container}>
+        <IndexHeader />
```

Comment:
Uh, where's `tourName`?

Reply:
hm, chris made the changes to this file. i doubt it needed to be touched at all. not sure where it's being rendered (this is the IndexScreen inside /components)